.github/workflows/build.yml

```diff
@@ -135,6 +135,32 @@ jobs:
       - run: yarn test:e2e:ganache:ws:${{ matrix.browser }}
         shell: bash
 
+  black-box:
+      name: Black Box
+      needs: build
+      runs-on: ubuntu-latest
+      env:
+        INFURA_HTTP: ${{ secrets.INFURA_HTTP }}
+        INFURA_WS: ${{ secrets.INFURA_WS }}
+        INFURA_WSS: ${{ secrets.INFURA_WSS }}
+        INFURA_GOERLI_WS: ${{ secrets.INFURA_GOERLI_WS }}
+        MODE: ${{ matrix.mode }}
+      strategy:
+        fail-fast: false
+        matrix:
+          node: [14, 16]
+          mode: ['http', 'ws']
+          backend: ['ganache', 'geth', 'infura']
```

Comment:
We already have multiple eth clients tests in e2e so for blackbox we can only test on one eth client and or Infura in github actions,  ( and should keep other Eth clients blackbox tests in scripts for local testing. )